Wrong example code which doesn't compile

### Page(s) Affected
<!--
Please list the page or pages this bug is present on. Please file one issue
for each bug you find. Only file an issue for a multi-page bug if the bug is
indeed the *same* bug, across the pages.
-->
https://www.rust-lang.org/learn/get-started
https://www.rust-lang.org/zh-CN/learn/get-started
https://www.rust-lang.org/zh-TW/learn/get-started
etc. (in other languages)
### What needs to be fixed?
<!--
Tell us what is causing the issue and (if possible) *how* it is causing the
issue.  For example, "The lowercase g (in 'Getting') overlaps with the l
(in 'development').".
-->
In the referred pages, the example code below the title *A small Rust application* DOESN'T compile.
```
error[E0308]: mismatched types
  --> src/main.rs:10:9
   |
10 |     say(message.as_bytes(), width, &mut writer).unwrap();
   |     --- ^^^^^^^^^^^^^^^^^^ expected `&str`, found `&[u8]`
   |     |
   |     arguments to this function are incorrect
   |
   = note: expected reference `&str`
              found reference `&[u8]`
note: function defined here
  --> /Users/username/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ferris-says-0.3.1/src/lib.rs:84:8
   |
84 | pub fn say<W>(input: &str, max_width: usize, mut writer: W) -> Result<()>
   |        ^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `rust-journey` (bin "rust-journey") due to previous error
```
### Suggested Improvement
<!--
(optional) Do you have a suggestion for how to fix the problem? Include your
ideas here.
-->
Fix the code with the following:
```rust
use ferris_says::say; // from the previous step
use std::io::{stdout, BufWriter};

fn main() {
    let stdout = stdout();
    let message = String::from("Hello fellow Rustaceans!");
    let width = message.chars().count();

    let mut writer = BufWriter::new(stdout.lock());
    say(&message, width, &mut writer).unwrap();        // Change only this line
}
```

--- a/templates/learn/get-started.html.hbs
+++ b/templates/learn/get-started.html.hbs
@@ -71,7 +71,7 @@ Hello, world!</code></pre>
     {{#fluent "learn-dependencies-steps"}}
     {{#fluentparam "cargotoml"}}
       <pre><code>[dependencies]
-ferris-says = "0.2"</code></pre>
+ferris-says = "0.3.1"</code></pre>
     {{/fluentparam}}
     {{/fluent}}
 
@@ -95,21 +95,19 @@ fn main() {
     let width = message.chars().count();
 
     let mut writer = BufWriter::new(stdout.lock());
-    say(message.as_bytes(), width, &mut writer).unwrap();
-}
-    </code></pre>
+    say(&message, width, &mut writer).unwrap();
+}</code></pre>
     {{/fluentparam}}
     {{#fluentparam "output"}}
-    <pre><code class="plaintext">----------------------------
+    <pre><code class="plaintext"> __________________________
 < Hello fellow Rustaceans! >
-----------------------------
-              \
-               \
-                 _~^~^~_
-             \) /  o o  \ (/
-               '_   -   _'
-               / '-----' \
-    </code></pre>
+ --------------------------
+        \
+         \
+            _~^~^~_
+        \) /  o o  \ (/
+          '_   -   _'
+          / '-----' \</code></pre>
     {{/fluentparam}}
     {{/fluent}}
 

